tfhe/src/integer/server_key/radix_parallel/scalar_comparison.rs

```diff
@@ -474,7 +470,7 @@ impl ServerKey {
             match self.is_scalar_out_of_bounds(lhs, rhs) {
                 Some(std::cmp::Ordering::Greater) | Some(std::cmp::Ordering::Less) => {
                     // Scalar is not within bounds so its not equal
-                    return self.create_trivial_radix(1, lhs.blocks().len());
+                    return BooleanBlock::new_unchecked(self.key.create_trivial(1));
```

Comment:
new trivial boolean rather (I think you added that function) ? I think there are quite a bit of new_unchecked on trivial 1s and 0s